docs/multiarch.rst

```diff
@@ -711,56 +711,62 @@ This mechanism can also be used to temporarily disable a worker
 cluster by removing it from the list or adding ``enabled: false`` to
 the cluster description for each platform.
 
-Annotations on orchestrator build
----------------------------------
+Annotations/labels on orchestrator build
+----------------------------------------
 
 The orchestrator build will fetch annotations from completed worker
 builds and add them to its own annotations to aid metrics
 reporting. The annotations will look as follows::
 
   metadata:
-  annotations:
-    worker-builds:
-      x86_64:
-        build:
-          cluster-url: openshift_url of worker cluster
-          namespace: default
-          build-name: repo-branch-abcde-1
-        digests:
-        - registry: ...
-          repository: ...
-          tag: ...
-          digest: ...
-        ...
-        plugins-metadata:
-          timestamps:
-            koji: ...
-            ...
-          durations:
-            koji: ...
-            ...
-          errors: {}
-      ppc64le:
-        build:
-          cluster-url: openshift_url of worker cluster
-          namespace: default
-          build-name: repo-branch-abcde-1
-        digests:
-        - registry: ...
-          repository: ...
-          tag: ...
-          digest: ...
-        ...
-        plugins-metadata:
-          timestamps:
-            koji: ...
-            ...
-          durations:
-            koji: ...
-            ...
-          errors: {}
-    plugins-metadata: '{"timestamps": {"orchestrate_build": "...", ...},
-      "durations": {"orchestrate_build": ..., ...}, "errors": {}}'
+    labels:
+      koji-build-id: ...
+    annotations:
```

Comment:
Updated to include that. While doing so, I noticed we also want `digests` (for sites without Koji integration).